camera-viewfinder-compose/build.gradle.kts

```diff
@@ -61,6 +61,8 @@ dependencies {
     implementation(composeBom)
     androidTestImplementation(composeBom)
 
+    implementation("androidx.compose.foundation:foundation:1.6.0-SNAPSHOT")
```

Comment:
This is fine. Just curious, is there a way to make foundation work with composeBom?